archive_communication/api-requirements.md

```diff
@@ -0,0 +1,324 @@
+# API Requirements
+
+This document outlines some general requirements for APIs that would be necessary
+for communication between BCDC, data archives, and other partners for one- or
+two-way communication. These requirements don't mandate a specific implementation, 
+but do make some assumptions around higher level design:
+
+## Assumptions
+- This document and the examples make a few assumptions about the APIs:
+    - APIs communicate via JSON over HTTP
+    - APIs roughly follow REST principles
+
+## General Requirements
+
+<table class="tg">
+<thead>
+  <tr>
+    <th class="tg-g7sd">#</th>
+    <th class="tg-g7sd">Requirement</th>
+    <th class="tg-g7sd">Description</th>
+    <th class="tg-g7sd">Priority</th>
+    <th class="tg-g7sd">Notes</th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td class="tg-lboi">1</td>
```

Comment:
@pbishwakarma the data/metadata we are making public through our different APIs is public, so no user authentication will be needed -at least for now-.